apps/prairielearn/src/ee/lib/lti13.sql

```diff
@@ -8,3 +8,89 @@ SELECT
     WHERE
       course_instance_id = $course_instance_id
   );
+
+-- BLOCK update_token
+UPDATE lti13_instances
+SET
+  access_tokenset = $access_tokenset,
+  access_token_expires_at = $access_token_expires_at
+WHERE
+  id = $lti13_instance_id;
+
+-- BLOCK update_lineitem
```

Comment:
Nit: this query name reflects the old table name. This should be named something like `upsert_lti13_assessment`. This can be addressed in a future PR.

Reply:
Yeah, that's reasonable. Easy enough to fix, so I did.